README.md

```diff
@@ -43,6 +43,10 @@ You can add them to your project via `idf.py add-dependancy`, e.g.
 
 Alternatively, you can create `idf_component.yml` file manually, such as in [this example](examples/display/main/idf_component.yml).
 
+### Recommendation for custom projects
+
+When you want to use the BSP for the real project, there is highly recommended disable the configuration option `CONFIG_BSP_ERROR_CHECK` in menuconfig and you should check all returned error codes from all called functions. Otherwise, when the option `CONFIG_BSP_ERROR_CHECK` is enabled, all errors in the BSP will break the program.
```

Comment:
```suggestion
When you want to use a BSP for real project, it is highly recommended to disable configuration option `CONFIG_BSP_ERROR_CHECK` in menuconfig. You should check all returned error codes from all called functions. Otherwise, when the option `CONFIG_BSP_ERROR_CHECK` is enabled, all errors in the BSP will abort the program.
```